src/apps/eth/eth_params.c

```diff
@@ -10582,6 +10582,13 @@ static const app_eth_erc20_params_t _ethereum_erc20_params[] = {
             "\x28\xde\xe0\x1d\x53\xfe\xd0\xed\xf5\xf6\xe3\x10\xbf\x8e\xf9\x31\x15\x13\xae\x40",
         .decimals = 18,
     },
+    {
+        .name = "Uniswap",
+        .unit = "UNI",
```

Comment:
The list seems to be sorted in alphabet order by the unit. I'd keep the order.

Reply:
done